spec/api/v1/advocacy_campaigns_spec.rb

```diff
@@ -33,6 +33,21 @@
         expect(api_advocacy_campaign).to eq(serialized_advocacy_campaign)
       end
     end
+
+    describe "GET /v1/advocacy_campaigns?filter[origin_system]" do
+      it "filters advocacy campaigns by origin_system" do
+        FiveCalls_event = create(:advocacy_campaign, title: "Event1", description: "Description1" , origin_system: "5calls", action_type: "phone")
+        Facebook_event = create(:advocacy_campaign, title: "Event2", description: "Description2" , origin_system: "Facebook", action_type: "in-person")
```

Comment:
FWIW, we don't actually scrape Facebook (and it's not really on the horizon for this project).